src/components/PurchaseOrder/PO.js

```diff
@@ -59,6 +64,16 @@ class PO extends Component {
         },
       },
     },
+    linesLimit: {
+      type: 'okapi',
+      records: 'configs',
+      path: 'configurations/entries',
+      GET: {
+        params: {
+          query: `(module=${MODULE_ORDERS} and configName=poLines-limit)`,
```

Comment:
please use constant for such things as config names, it's also expected to use in several places